doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -0,0 +1,667 @@
+---
+slug: /poscreators/middleware-doc/digital-receipt/implementation/digital-receipt-implementation
+title: 'Digital receipt implementation'
+---
+
+# Digital receipt transmission
+
+>[!IMPORTANT]
```

Comment:
I don't think this will render correctly on the docs page, better to use admonitions:
https://docusaurus.io/docs/markdown-features/admonitions